doc/UsersGuide/source/doc-snippets/hsd_data.rst

```diff
@@ -0,0 +1,6 @@
+Data for the HSD cases is already staged on Tier-1 platforms at the ``INPUTROOT_*`` locations listed in `baseline_setup.yaml <https://github.com/ufs-community/ufs-weather-model/blob/develop/tests-dev/baseline_setup.yaml>`_. However, users on any platform can download the data directly from the `HTF data bucket <https://registry.opendata.aws/noaa-ufs-htf-pds/>`_ using ``wget``. 
+
+.. code-block:: console 
+
+   wget https://noaa-ufs-htf-pds.s3.amazonaws.com/develop-20241025/HSD_INPUT_DATA/HSD_INPUT_DATA.tar.gz
```

Comment:
please change this link to 

https://noaa-ufs-htf-pds.s3.amazonaws.com/develop-20241115/HSD_cases_data.tar.gz

Reply:
What's the difference between this link and https://noaa-ufs-htf-pds.s3.amazonaws.com/develop-20241115/HSD_cases_data.tar.gz ?

Reply:
https://noaa-ufs-htf-pds.s3.amazonaws.com/develop-20241025/HSD_INPUT_DATA/ was uploaded earlier and prior to input data fixes. we need the 20241115 dated one.